src/scss/grommet-core/_objects.paragraph.scss

```diff
@@ -28,4 +28,9 @@
       font-weight: 600;
     }
   }
+
+  &.paragraph--width-large {
+    width: $inuit-base-spacing-unit * 30;
```

Comment:
Assuming we will convert to the standard sizing variables when the Box sizing pull request comes in.